osu.Game.Rulesets.Osu/Edit/PreciseRotationPopover.cs

```diff
@@ -59,13 +61,17 @@ private void load()
                             new RadioButton("Playfield centre",
                                 () => rotationInfo.Value = rotationInfo.Value with { Origin = RotationOrigin.PlayfieldCentre },
                                 () => new SpriteIcon { Icon = FontAwesome.Regular.Square }),
-                            new RadioButton("Selection centre",
+                            selectionCentreButton = new RadioButton("Selection centre",
                                 () => rotationInfo.Value = rotationInfo.Value with { Origin = RotationOrigin.SelectionCentre },
                                 () => new SpriteIcon { Icon = FontAwesome.Solid.VectorSquare })
                         }
                     }
                 }
             };
+            selectionCentreButton.Selected.DisabledChanged += isDisabled =>
+            {
+                selectionCentreButton.TooltipText = isDisabled ? "Select more than one circles to perform rotation." : string.Empty;
```

Comment:
probably objects makes more sense.

Reply:
Not really, a slider is an object, but still can be rotated around itself.

Reply:
Except that selecting a circle and a slider also works, so I think "object" makes more sense.

Reply:
We're glossing over some edge cases like selecting a spinner and a circle is technically multiple objects, but still cant be rotated around themselves.

But I mean, it's straight-forward enough so it's ok.

Reply:
> selecting a spinner and a circle is technically multiple objects, but still cant be rotated around themselves

Will probably be not a problem when https://github.com/ppy/osu/issues/7003 is eventually implemented (i.e. spinner selection area will probably be the size of a hit circle).